constellations/gemini-user/src/afh/afh.sass

```diff
@@ -0,0 +1,46 @@
+@import "../utils/constants.sass"
+
+#view-afh
+    width: 70%
+    margin: 0 auto
+    font-family: $fontFamilyRoboto
+
+    &  h1
+        margin: $spacing6x 0
+        color: $colorTurquoise
+    
+    .description
+        margin: $spacing2x 0
+        color: $colorBlack
+
+
+    .tab 
+        overflow: hidden
+        border-bottom: 1px solid $colorGrayLight
+
+    .tabButton 
+        float: center
+        width: 33.333%
+        cursor: pointer
+        padding: 14px 16px
+        transition: 0.3s
+        background-color: $colorWhite
+        font-size: 17px
+        color: $colorBlack
+
+    .tab button:hover
+        cursor: pointer
+        background-color: $colorGrayLight
+    .tab button:active
+        background-color: $colorTurquoise
+    
```

Comment:
i'm having a bit of trouble here. when i select on a tab, i want the background color to change (will change it to underline later). however the background only turns turquoise when i'm holding down on the button. after, i let go the color is back to white. idk if you guys have any ideas @notewio @ahsu1230

Reply:
I think you have to do this with js, maybe instead of `.tab button:active` have something like `.tab button.active` and then set the classes on the buttons accordingly